### ISSUE TITLE
SW-1000104

### ISSUE THREAD

**Issue by aizaz-clym** [aizaz-clym/clym-test#4]
## Headings and labels are semantically correct

| Type | Standard | Category | Occurrences | Page | Scanned | Status |
| --- | --- | --- | --- | --- | --- | --- |
| Automatic | WCAG | System | 1 | Home | 2026-07-15 | 🔴 Failed |

### Steps

1. Find all the headings and labels.
2. Make sure that headings have text and are used in a hierarchical way that is readable by AT.
3. Make sure labels have text and are assigned to fields.

### Criteria

| Criteria | Name | Description | Level |
| --- | --- | --- | --- |
| [2.4.6](https://www.w3.org/WAI/WCAG22/quickref#headings-and-labels) | Headings and Labels | Headings and labels describe topic or purpose. | AA |

### Selector

`html`

### HTML

```html
<html lang="en">
```

<details>
<summary>Remediation</summary>

#### Description

Make section headings descriptive, helping users find specific content and orient themselves within the page. Consider placing the most important info at the beginning of each heading to help users skim headings effectively when navigating from heading to heading with browsers or AT.

#### Category: General

| Criteria | Name | Description | Level |
| --- | --- | --- | --- |
| [2.4.6](https://www.w3.org/WAI/WCAG22/quickref#headings-and-labels) | Headings and Labels | Headings and labels describe topic or purpose. | AA |

<details>
<summary>Examples</summary>

##### Example 1: An HTML page that describes the range of tasks for disaster preparation

```html
<h1>Disaster preparation</h1>
 <h2>Flood preparation</h2>
 <h2>Fire preparation</h2>
```

</details>

</details>

---

> Created automatically by [Clym](https://clym.io)
